common/types/mapper/thrift/shared_test.go

```diff
@@ -1102,7 +1102,10 @@ func TestDescribeTaskListResponseConversion(t *testing.T) {
 	for _, original := range testCases {
 		thriftObj := FromDescribeTaskListResponse(original)
 		roundTripObj := ToDescribeTaskListResponse(thriftObj)
-		assert.Equal(t, original, roundTripObj)
+		opt := cmpopts.IgnoreFields(types.DescribeTaskListResponse{}, "PartitionConfig")
```

Comment:
why are we ignoring PartitionConfig field?

Reply:
Only GRPC type has this field. The thrift type doesn't have this field